dask_cuda/disk_io.py

```diff
@@ -125,11 +125,11 @@ def __init__(
 
         if self.gds_enabled:
             try:
-                import cucim.clara.filesystem as cucim_fs  # noqa F401
+                import kvikio  # noqa F401
             except ImportError:
                 raise ImportError("GPUDirect Storage requires the cucim Python package")
             else:
-                self.gds_enabled = bool(cucim_fs.is_gds_available())
+                self.gds_enabled = kvikio.DriverProperties().is_gds_availabe
```

Comment:
Looks like there is a typo in the property. Submitted PR ( https://github.com/rapidsai/kvikio/pull/78 ) to address